src/types/app-types.ts

```diff
@@ -1,30 +1,22 @@
-export interface Timer {
-  title: string;
-  game: string;
-  initialTime: string;
-  timeAtPause: string
-}
+import { GameKey } from "../components/Timer/timerStyles";
+import { UseTimerOptions } from "../components/Timer/useTimer";
 
-export interface IncomingTimerProps {
+export interface TimerConfig {
```

Comment:
there was a couple types trying to define these concerns. one had name, the other had title. deduped this to clean stuff up a bit